backend/muster-roll/muster-roll-v2-api-swagger-2.0.0.yaml

```
@@ -0,0 +1,1048 @@
+    String version = versionHelper.getMusterRollVersion(musterRoll); // "V1" or "V2"
+    ```
+
+    ## Migration Path
+
+    **V1 to V2 Migration**:
+    1. Existing V1 projects continue working (no action required)
+    2. Create billing configuration in expense-calculator service
+    3. New muster rolls automatically use V2 mode with period linkage
+    4. Old V1 muster rolls remain unchanged (billingPeriodId=NULL)
+
+  version: 2.0.0
```

YAMLlint (1.37.1)

[error] duplication of key "version" in mapping

(key-duplicates)